docs/ref/models/conditional-expressions.txt

```diff
@@ -209,6 +209,12 @@ What if we want to find out how many clients there are for each
     >>> Client.objects.aggregate(
     ...     regular=Count('pk', filter=Q(account_type=Client.REGULAR)),
     ...     gold=Count('pk', filter=Q(account_type=Client.GOLD)),
-    ...     platinum=Count('pk', filter=Q(account_type=Client.PLATINUM)
+    ...     platinum=Count('pk', filter=Q(account_type=Client.PLATINUM)),
     ... )
     {'regular': 2, 'gold': 1, 'platinum': 3}
+
+
+On backends that support it the `SQL 2003 FILTER WHERE`_ syntax is used, otherwise
+this is emulated using a ``CASE`` statement.
+
+.. _`SQL 2003 FILTER WHERE`: http://modern-sql.com/feature/filter
```

Comment:
I think the policy is not to include links to third-party pages, but only to official documentation or Wikipedia, but the closest thing in the SQL:2003 article was a highlight about increased OLAP functionality.

Perhaps a SQL example is better to illustrate what it translates into if the SQL:2003 filter is implemented?